<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(138, 63, 137);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEquals">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button0">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMinus">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPlus">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 18pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDivide">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMultiply">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(130, 0, 195);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
